### PR TITLE
update github page with detailed example

### PR DESCRIPTION
**Pull request by RonHildebrandt**
An updates to a more detailed use example, directly in front page. Could help people to test it at all, especially people who dont deal at all with software development/GitHub.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,5 +32,57 @@ within the field of raman into a standardized representation using the
 ## Docs
 Extensive documentation of this pynxtools plugin is available [here](https://fairmat-nfdi.github.io/pynxtools-raman/). You can find information about getting started, how-to guides, the supported file formats, how to get involved, and much more there.
```
Comment (lukaspie) [inline README.md:33]: This link does not exist (yet).

**Comment by lukaspie** [comment]
Have you considered making this a full-blow mkdocs documentation? All of our other plugins have this. If not, you should at least remove the link that I highlighted here.

**Comment by RonHildebrandt** [comment]
@lukaspie I removed the incorrect link (as this was just copied over from pynxtools-ellips)
Dont want to put in a extensive doc right now. Think the small hints in the main README file are enough for now.

File: README.md [hunk 2]
```diff
@@ -29,8 +29,57 @@ This reader plugin for [`pynxtools`](https://github.com/FAIRmat-NFDI/pynxtools)
...
+Download the repository via git clone:
+```shell
+git clone https://github.com/FAIRmat-NFDI/pynxtools-raman.git
+```
+Switch to the project root folder:
+```shell
+cd pynxtools-raman
+```
+You see 3 Folders:
+- examples: contains example datasets, to show how the data conversion is done (currently 1 example from WITec and 1 example from the Raman Open Database)
```
Comment (lukaspie) [inline README.md:43]:
```suggestion
You see three folders:
- examples: contains example datasets, to show how the data conversion is done (currently one example from WITec and one example from the Raman Open Database)
```
Write out numbers one through nine

File: README.md [hunk 3]
```diff
@@ -29,8 +29,57 @@ This reader plugin for [`pynxtools`](https://github.com/FAIRmat-NFDI/pynxtools)
...
+- tests: contains a test procedure and files, which are required for software development
+- src/pynxtools_raman: contains the source files, which contain the sub-reader function for Raman experiments. This only works in combination with the Python package [pynxtools](https://github.com/FAIRmat-NFDI/pynxtools). This contains the [Multiformat Reader](https://fairmat-nfdi.github.io/pynxtools/how-tos/use-multi-format-reader.html) and the respective sub-reader functions for WITec or the Raman Open Database. This also contains the config.json files in src/pynxtools_raman/config, which are necessary to map the data via the Multiformat Reader, by as well allowing individual adjustments. In this way each laboratory is able to map the data via the same reader, while each laboratory has its own individual electronic lab notebook structure.
```
Comment (lukaspie) [inline README.md:45]: The MultiFormatReader is not contained within src/pynxtools_raman, only a reader that inherits from it

File: README.md [hunk 4]
```diff
@@ -29,8 +29,57 @@ This reader plugin for [`pynxtools`](https://github.com/FAIRmat-NFDI/pynxtools)
...
+
+Consider setting up an invididual python environment, to seperate the python fuctnionalities of this package from the python funtionalities of your operating system:
```
Comment (lukaspie) [inline README.md:47]:
```suggestion
Consider setting up an individual python environment to seperate the python functionalities of this package from the python funtionalities of your operating system:
```

Comment (lukaspie) [inline README.md:47]: I would just put a link to resources describing venvs in python, see https://github.com/FAIRmat-NFDI/pynxtools#installation

File: README.md [hunk 5]
```diff
@@ -29,8 +29,57 @@ This reader plugin for [`pynxtools`](https://github.com/FAIRmat-NFDI/pynxtools)
...
+```
+
+and for the Raman Open Database dataset set via:
+```shell
+dataconverter examples/database/rod/rod_file_1000679.rod src/pynxtools_raman/config/config_file_rod.json --reader raman --nxdl NXraman --output new_rod_example_neuxs.nxs
+```
+
+**For Example for the Raman Open Database command:**
+- You assign the reader name via `--reader raman`.
+- You assign the NeXus definition language (nxdl) via `--nxdl NXraman`.
```
Comment (lukaspie) [inline README.md:77]: You don't assing the NeXus definition language, you define which application definition that is _described_ in nxdl to use.

File: README.md [hunk 6]
```diff
@@ -29,8 +29,57 @@ This reader plugin for [`pynxtools`](https://github.com/FAIRmat-NFDI/pynxtools)
...
+- You specify the name and path of the output file via `--output new_rod_example_neuxs.nxs`.
+- You assign an individualized config file via `src/pynxtools_raman/config/config_file_rod.json`. The config file is detected by its extension `.json`.
+- You give the file, which includes the meta and measurement data via `examples/database/rod/rod_file_1000679.rod`. The parser is specified to detect the `.rod` file, and handle the content appropriately.
```
Comment (lukaspie) [inline README.md:80]:
```suggestion
- You give the file which includes the meta and measurement data via `examples/database/rod/rod_file_1000679.rod`. The parser is specified to detect the `.rod` file, and handle the content appropriately.
```

**Review by lukaspie — APPROVED** [review]
LGTM, except for the points outlined below. I still think that this should be placed in a how-to in the docs, but fine for now.

**Review by lukaspie — APPROVED** [review]
LGTM, except for the points outlined below. I still think that this should be placed in a how-to in the docs, but fine for now.

File: README.md (hunk 5 above)
Comment (RonHildebrandt) [reply README.md:77]: ah yea, all right. Adjusted it.

File: README.md (hunk 2 above)
Comment (RonHildebrandt) [reply README.md:43]: done

File: README.md (hunk 4 above)
Comment (RonHildebrandt) [reply README.md:47]: restructured this part.

Comment (RonHildebrandt) [reply README.md:47]: I put a link in addition to this - anyway want to keep this short how to for this.

File: README.md (hunk 6 above)
Comment (RonHildebrandt) [reply README.md:80]: done

**Pull request merged**